lib/chat/key_share.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Chat.KeyShare do
+  @moduledoc "Manipulate social sharing keys"
+
+  alias Chat.Identity
+  alias Chat.{Actor, ChunkedFiles, ChunkedFilesMultisecret}
+  alias Chat.Upload.UploadKey
+  alias Chat.{Dialogs, Dialogs.Dialog, Messages}
+
+  alias ChatWeb.MainLive.Page
```

Comment:
`Chat` should not use `ChatWeb`

Reply:
changed to use Phoenix.PubSub for broadcasting